changelog/2024-10-17-introducing-conduit-operator.md

```diff
@@ -0,0 +1,18 @@
+---
+slug: '2024-10-17-conduit-operator'
+title: Introducing Conduit Operator
+draft: false
+tags: [conduit, scale, kubernetes]
+---
+
+We're happy to announce that we recently released [Conduit Operator v0.0.1](https://github.com/ConduitIO/conduit-operator/releases/tag/v0.0.1), a Kubernetes operator that extends the Kubernetes API to enhance the management of Conduit instances on our own [Conduit Platform](https://meroxa.io).
```

Comment:
We have v0.0.2 release which adds schema support https://github.com/ConduitIO/conduit-operator/releases/tag/v0.0.2

Do you want add another changelog entry?

Reply:
@lyuboxa this was imagining this was actually announced when it was launched.

Reply:
Added a new changelog for this https://github.com/ConduitIO/conduit-site/pull/173/commits/a969be12b9604b4d91f23ecc3f9c511def7f6630